README.md

```diff
@@ -38,23 +44,40 @@ Oh.  In case it's not clear: it's a hard limit on 10 minutes.  If you need more
 * We may occassionally hit a cut off and need to reschedule you for the next session
 * We'll include an order-of-operations and indicate demos that may be at risk of being rescheduled
 
+### How are talks scheduled?
+
```

Comment:
lgtm!  Only question!  To me (and I might have a different take than others) randomized means 'we'll definitely do this in a random way,' vs. us just moving things around a little?

e.g. for 8-10 talks, that might mean `get all open proposals`+`randomly pick 10`, vs. `get 10 most recent`, or `get 12 most recent`+`randomly pick 10`

Similar for order - if the order we receive demos in works, do we want to intentionally randomize the order of selected demos, or have some control over optimization (e.g. the issue create date did this for us, but putting Doug's intro to ImportExcel ahead of a talk that uses ImportExcel among other schedule tweaks seems reasonable)?

Cheers!

Reply:
Good point - semi-randomized? Randomized prior to thematic optimization? Not sure on language here, definitely worth thinking about.

Reply:
maybe something like `we'll try to randomize`?  Less onus on us to use some randomization algorithm : P  we could clarify what we mean once we have better phrasing / idea of how we'll do it?

Reply:
works for me!

Reply:
@RamblingCookieMonster Updated!